easybuild/easyconfigs/c/cdat/cdat-8.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,132 @@
+easyblock = 'PythonBundle'
+name = 'cdat'
```

Comment:
name should be `CDAT` according to description?

Reply:
Yup, old name from when I took over the EC